### ISSUE TITLE
Upgrade: UI displays snapshot creation time as zero after upgrade

### ISSUE THREAD

**Issue by soumyalj** [rancher/rancher#18760]
Version: Upgrade from v2.1.7 to v2.2.0-rc3
K8s version: v1.13.4

Steps:
1. Create a DO cluster in v2.1.7 with "Creation interval period" as 15m and "creation retention period" as 1h
2. Create few workloads/ingress
3. Upgrade rancher to v2.2.0-rc3

After upgrade, when we edit the cluster, snapshot creation period is shown as zero hours. It should show a default of 12h

![Screen Shot 2019-03-09 at 9 41 30 AM](https://user-images.githubusercontent.com/18536626/54075224-bb14d680-4251-11e9-91fc-3a3ab15a636e.png)

**Comment by cjellick** [comment]
should we indicate on cluster list page that a cluster is using the old style snapshot and they should switch to new version

**Comment by westlywright** [comment]
Per discussion with qa the setting should round to the nearest whole hour. 15m -> 1 hour

**Comment by davidnuzik** [comment]
Version: v2.1.7 -> master (upgraded to master so we get latest UI and thus fix)
K8s version: v1.13.4

Executed steps as defined in issue description. Also created a second cluster with only 2m snapshot interval time and 45m retention period. I waited for a while so we take a few snapshots on second cluster (2m cluster).

Confirmed that BOTH rounded up to nearest hour for "Snapshot Creation Period" time upon upgrade as expected with the fix.
Note, default recurring etcd Snapshot Retention Count is always 6.

![image](https://user-images.githubusercontent.com/45179589/54221620-201a3780-44b1-11e9-88a9-1130d1ba9ceb.png)

Smoke tested both. I did not identify any issues.
